loops-and-arrays.js

```diff
@@ -35,8 +66,24 @@ const numbers = [2, 4, 5, 29, 38];
 // Zorg ervoor dat dit ook zou werken als de array wel 100 of 200 getallen zou bevatten!
 // ==========================================
 
+//stappenplan
+// maak een nieuwe array met de waardes van squares, noem deze volume
+//print de originele volume array
+//roep de waardes van volume array een voor een op
+//bereken het volume (lengte * breedte * hoogte) met de opgevraagde waarde
+// waarde[1] * waarde[1] * waarde[1]
+//herhaal dit voor de hele array
+
 const squares = [30, 2, 8, 24, 11];
+//maak een nieuwe array aan met dezelfde getallen.
+let volume = squares;
```

Comment:
Het is niet perse nodig om een let aan te maken. Je kan ook zonder de let de opdracht prima voltooien.

```suggestion
for (let i = 0; i < squares.length; i++) {
  squares[i] = squares[i] * squares[i] * squares[i];
}

console.log(squares);
```